hack/quickstart/init-master.sh

```diff
@@ -62,6 +62,9 @@ function init_master_node() {
         --mount volume=home,target=/core \
         --net=host ${BOOTKUBE_REPO}:${BOOTKUBE_VERSION} --exec \
         /bootkube -- start --asset-dir=/core/assets
+
+    # Add auto-approver asset.
+    curl --silent -H "Content-Type: application/yaml" -XPOST -d"$(cat /home/core/assets/extra/kubelet-approver.yaml)" "http://127.0.0.1:8080/apis/extensions/v1beta1/namespaces/kube-system/deployments" > /dev/null
```

Comment:
Can we move this to `hack/tests/conformance-gce/sh` right between `add_masters` and `add_workers` being called around like 93.